src/progress-indicator/progress-indicator.component.ts

```diff
@@ -28,7 +28,7 @@ import { Step } from "./progress-indicator-step.interface";
 			class="bx--progress-step bx--progress-step--{{step.state[0]}}"
 			*ngFor="let step of steps; let i = index"
 			[ngClass]="{'bx--progress-step--disabled' : step.disabled}">
-			<div class="bx--progress-step-button bx--progress-step-button--unclickable" role="button" tabindex="-1">
+			<button class="bx--progress-step-button bx--progress-step-button--unclickable" role="button" tabindex="-1">
```

Comment:
Converted to `button` to fix gap btw text and optionalText. (react has button)
<img width="168" alt="Screenshot 2023-04-07 at 2 51 54 PM" src="https://user-images.githubusercontent.com/14229415/230663660-93cf2e3b-4f01-4384-96a2-6a7dde90dbc3.png">